PRESUBMIT.py

```diff
@@ -33,7 +33,10 @@ def _web_files_filter(affected_file):
             files_to_check=[
                 r'.+\.(js|jsx|ts|tsx|css|less|lss|sass|scss|svelte)$',
                 r'package\.json$'
-            ])
+            ],
+            files_to_skip=input_api.DEFAULT_FILES_TO_SKIP + (
```

Comment:
I think this won't actually fix the problem - the audit tokens thing runs over all files and doesn't take this skip list into account. Maybe we need to be able to pass these through to ~~Leo~~ Nala

Reply:
Yeah that makes complete sense